### PR TITLE
Move installation.txt to INSTALL.md

### PR DESCRIPTION
**Pull request by cpeel**
Move the installation instructions to `INSTALL.md` and split out the
upgrade instructions into `UPGRADE.md`. Add a `README.md` to help people
navigate the directory's contents (at least the documentation).

You can view rendered versions of the above here on github starting at https://github.com/cpeel/dproofreaders/tree/installation-md/SETUP

### CODE REVIEW

File: SETUP/INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+security measure, move it out of your webserver's document hierarchy.**
+Don't delete it as you may want to use its contents again to
+reconfigure the site.
+
+### At this point, your DP site should be functional.
+Try visiting it via your browser.
+(You may want to redirect/rewrite requests for foo/ to foo/c/.
+You could just create a foo/index.html that redirects.)
+Register as a new user.
+Of course, there aren't any projects to proofread yet.
```
Comment (srjfoo) [inline SETUP/INSTALL.md]: This doesn't seem to be a particularly useful statement. Does it even need to be there? If projects even need to be mentioned at this point, perhaps this should point to a document that describes creating a project? However, a bit down from here, defining a siteadmin is described; is that the purpose of registering this account?

File: SETUP/UPGRADE.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+```bash
+mv c c.old
+```
+
+## Unpack the new DP code
+Unpack the new DP code using either `tar -xf` or `unzip`. This creates
+a new 'c' hierarchy.
+
+## Configure the new DP code
+Follow the 'Installing from scratch' instructions in [README.md](README.md).
```
Comment (srjfoo) [inline SETUP/UPGRADE.md]: README.md or INSTALL.md?

File: SETUP/INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+The following PHP extensions are required. They are listed below with their
+Ubuntu system package names.
+* GD - php-gd
+* Internationalization - php-intl
+* MySQL - php-mysql
+* zip - php-zip
+
+### MySQL
+MySQL version 5.5 or later is recommended.
+Versions 5.1 and later may work but are untested.
```
[the remark this answers is not in the document]
Comment (cpeel) [reply SETUP/INSTALL.md]: Good suggestion, updated.

File: SETUP/INSTALL.md [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+
+vim wikihiero/wikihiero.php
+# On line 59, for definition of WH_IMG_DIR,
+# change "$wgScriptPath/extensions/wikihiero/img/"
+# to just "img/"
+
+vim wikihiero/wh_language.php
+# The file begins with three bytes (EF BB BF), which
+# constitute the Unicode Byte Order Mark encoded in UTF-8.
+# Delete those bytes?
```
[the remark this answers is not in the document]
Comment (cpeel) [reply SETUP/INSTALL.md]: I'm going to punt on the WikiHiero section for the moment until we decide if we want to keep it at all. It's on my TODO list now though.

File: SETUP/INSTALL.md [hunk 5]
```diff
@@ -0,0 +1,389 @@
...
+DP tables with the same authentication information.
+
+See the "Post installation / conversion steps" section in
+`phpbb3-conversion.txt` for recommended phpBB3 configuration settings
+and further steps to ensure good integration with the DP code.
+
+### Create phpBB categories and forums
+
+When the site is operational, each team and each project will get an
+automatically-created discussion topic. You should create a forum to
```
[the remark this answers is not in the document]
Comment (cpeel) [reply SETUP/INSTALL.md]: I massaged the wording a bit, but I don't think the nuance is important for this particular part of the README.

File: SETUP/INSTALL.md [hunk 6]
```diff
@@ -0,0 +1,389 @@
...
+    d
+    d/locale
+    d/stats
+    d/stats/automodify_logs
+    d/teams
+    d/teams/avatar
+    d/teams/icon
+    d/pg
+    d/xmlfeeds
+    (You also need to put some files in d??)
```
[the remark this answers is not in the document]
Comment (cpeel) [reply SETUP/INSTALL.md]: I've removed the line. I'm fairly confident additional files are no longer required after I made the per-round image headings optional.

File: SETUP/INSTALL.md [hunk 7]
```diff
@@ -0,0 +1,389 @@
...
+### Move SETUP directory outside your webserver's document hierarchy
+The `c/SETUP/` directory is only needed for site configuration. **As a
+security measure, move it out of your webserver's document hierarchy.**
+Don't delete it as you may want to use its contents again to
+reconfigure the site.
+
+### At this point, your DP site should be functional.
+Try visiting it via your browser.
+(You may want to redirect/rewrite requests for foo/ to foo/c/.
+You could just create a foo/index.html that redirects.)
```
[the remark this answers is not in the document]
Comment (cpeel) [reply SETUP/INSTALL.md]: I think it's OK as-is.

File: SETUP/UPGRADE.md (hunk 2 above)
Comment (cpeel) [reply SETUP/UPGRADE.md]: Good catch - thanks! I originally had the file as `README.md` and realized that it really should be `INSTALL.md` and missed this.

File: SETUP/INSTALL.md (hunk 1 above)
Comment (cpeel) [reply SETUP/INSTALL.md]: Yeah, I don't think that's necessary. I've removed it.

**Comment by cpeel** [comment]
Thanks for catching the weird bold/italics rendering. The local tool that is suppose to render the markdown in github format didn't do that and I missed it.

**Pull request merged**